doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide.md

```diff
@@ -2,76 +2,77 @@
 
 **X-ROAD 7**
 
-Version: 2.52  
+Version: 2.53  
```

Comment:
Does changing the global configuration sign key type require changes to the Security Server's configuration? If it does, it should be explained in the Security Server user guide.

Reply:
Moved my changes to another PR dedicated for security server, but left version changes because change log states that this is current version